app/portainer/services/api/stackService.js

```diff
@@ -278,36 +300,49 @@ function StackServiceFactory($q, Stack, ResourceControlService, FileUploadServic
     return deferred.promise;
   };
 
-  service.createComposeStackFromGitRepository = function(name, repositoryOptions, env, endpointId) {
+  service.createStackFromGitRepository = function(stackDeployOptions, repositoryOptions) {
```

Comment:
See my comment above regarding the `createStackFromFileUpload` function, please revert to the old behavior.